.github/workflows/notify_on_pull_request_open/fixtures/pr-event-long-description.json

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
why is this file present on the PR ? 🤔

Reply:
We us this files to test the script. The new file has a very subtle [change](https://github.com/osscameroon/global-github-actions/blob/cca27918388b7077771b9963713bd2adcb069198/.github/workflows/notify_on_pull_request_open/fixtures/pr-event-long-description.json#L185)  that is difficult to spot.